app/Metadata/Geodecoder.php

```diff
@@ -85,7 +85,8 @@ public static function decodeLocation(?float $latitude, ?float $longitude): ?str
 	 */
 	public static function decodeLocation_core(float $latitude, float $longitude, ProviderCache $cachedProvider): ?string
 	{
-		$geocoder = new StatefulGeocoder($cachedProvider, Configs::get_value('lang'));
+		$lang = Configs::getValueAsString('lang') ?: null;
```

Comment:
```suggestion
		$lang = Configs::getValueAsString('lang') ?: null;
```

I wonder, if we should introduce `getValueAsNullableString` as well? I have seen this kind of construction some times now,

Reply:
I thought I had seen this constructions several times, but after a closer look I see that it does not. So closing this conversation.